rdtools/analysis_chains.py

```diff
@@ -138,7 +138,8 @@ def __init__(self, pv, poa_global=None, temperature_cell=None, temperature_ambie
 
     def set_clearsky(self, pvlib_location=None, pv_azimuth=None, pv_tilt=None,
                      poa_global_clearsky=None, temperature_cell_clearsky=None,
-                     temperature_ambient_clearsky=None, albedo=0.25):
+                     temperature_ambient_clearsky=None, albedo=0.25,
+                     solar_position_method=None):
```

Comment:
Let's be explicit about the default behavior, otherwise we rely on the default behavior of pvlib, which is in version 0, so could potentially change.

Reply:
Is the concern that the default algorithm might change and affect the analysis results?  My thought was that relying on the pvlib default is actually preferable; I think the default method name (e.g. `nrel_numpy`) is more likely to change than the default algorithm is, if I ever dust off https://github.com/pvlib/pvlib-python/pull/1098 anyway.

Reply:
Yep, the concern is silently changing the calculations under the hood.